src/ape/types/signatures.py

```diff
@@ -38,6 +41,12 @@ def encode_vrs(self) -> bytes:
     def encode_rsv(self) -> bytes:
         return _left_pad_bytes(self.r, 32) + _left_pad_bytes(self.s, 32) + to_bytes(self.v)
 
+    def messageHash(self) -> HexBytes:
+        return self._messageHash
+
+    def signature(self) -> HexBytes:
+        return self._signature
```

Comment:
we already have `encode_*` methods for properly encoding the signature for different applications

Reply:
These may need to be documented better to show how the encode methods can be used in smart contract applications

Reply:
how about we keep the both ? if people want to get the signature directly without calling any further functions?

Reply:
I don't think this particular function adds a lot of value, and also maybe confuse people who are seeking the encoded signatures (where it is useful to call out which way you are encoding)